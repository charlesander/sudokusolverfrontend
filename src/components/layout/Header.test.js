import React from "react";
import { create } from "react-test-renderer";
import Header from "./Header"

describe("Header component", () => {
    test("Matches the snapshot", () => {
        const header = create(<Header />);
        expect(header.toJSON()).toMatchSnapshot();
    });
})