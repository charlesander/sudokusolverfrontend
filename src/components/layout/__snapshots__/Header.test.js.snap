// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Matches the snapshot 1`] = `
<h1>
  Sudoku Solver
</h1>
`;
